lerna diff without package fails

## Expected Behavior
Diff all packages

## Current Behavior
```
> @unction/all@5.0.2 diff /Users/krainboltgreene/Code/krainboltgreene/unction.js
> lerna diff

/Users/krainboltgreene/Code/krainboltgreene/unction.js/node_modules/.bin/lerna diff <pkg>

Global Options:
  --loglevel                       What level of logs to report.                                                                                      [string] [default: info]
  --concurrency                    How many threads to use if lerna parallelises the tasks.                                                              [number] [default: 4]
  --scope                          Restricts the scope to package names matching the given glob.
                                   (Only for 'run', 'exec', 'clean', 'ls', and 'bootstrap' commands)                                                                  [string]
  --ignore                         Ignore packages with names matching the given glob.
                                   (Only for 'run', 'exec', 'clean', 'ls', and 'bootstrap' commands)                                                                  [string]
  --include-filtered-dependencies  Include all transitive dependencies when running a command, regardless of --scope or --ignore.
  --registry                       Use the specified registry for all npm client operations.                                                                          [string]
  --sort                           Sort packages topologically (all dependencies before dependents)                                                  [boolean] [default: true]
  --max-buffer                     Set max-buffer(bytes) for Command execution                                                                                        [number]
  -h, --help                       Show help                                                                                                                         [boolean]
  -v, --version                    Show version number                                                                                                               [boolean]

Not enough non-option arguments: got 0, need at least 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @unction/all@5.0.2 diff: `lerna diff`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @unction/all@5.0.2 diff script 'lerna diff'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the @unction/all package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     lerna diff
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs @unction/all
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls @unction/all
npm ERR! There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/krainboltgreene/.npm/_logs/2017-05-13T23_44_45_696Z-debug.log
```

<details><summary>lerna.json</summary><p>
<!-- browsers demand the next line be empty -->

```json
{
  "version": "independent",
  "lerna": "2.0.0-rc.4",
  "packages": [
    "source/*"
  ]
}
```
</p></details>

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 2.0.0-rc.4 |
| `npm --version`  | 4.5.0 |
| `node --version` | v7.7.2 |

--- a/src/commands/DiffCommand.js
+++ b/src/commands/DiffCommand.js
@@ -8,7 +8,7 @@ export function handler(argv) {
   return new DiffCommand([argv.pkg], argv).run();
 }
 
-export const command = "diff <pkg>";
+export const command = "diff [pkg]";
 
 export const describe = "Diff all packages or a single package since the last release.";
 
@@ -25,6 +25,9 @@ function getLastCommit(execOpts) {
 export default class DiffCommand extends Command {
   initialize(callback) {
     const packageName = this.input[0];
+
+    // don't interrupt spawned or streaming stdio
+    this.logger.disableProgress();
 
     let targetPackage;
 

